components/molecule/select/src/index.scss

```diff
@@ -0,0 +1,50 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '~@s-ui/react-atom-input/lib/index';
+@import '~@s-ui/react-molecule-input-tags/lib/index';
+@import '~@s-ui/react-molecule-dropdown-option/lib/index';
+@import '~@s-ui/react-molecule-dropdown-list/lib/index';
+
+$base-class: '.sui-MoleculeSelect';
+$class-input: '#{$base-class}-inputSelect';
+
+$z-select-list: $z-navigation !default;
+$mr-select-list-arrow: -34px !default;
+
+#{$base-class} {
+  position: relative;
+
+  .sui-MoleculeDropdownList {
+    position: absolute;
+    z-index: $z-select-list;
+  }
+
+  #{$class-input} {
+    &-container {
+      align-items: center;
+      border: 1px solid transparent;
+      cursor: pointer;
+      display: flex;
+    }
+
+    &-arrow {
+      margin-left: $mr-select-list-arrow;
+
+      &--up {
+        transform: rotate(180deg);
+      }
+    }
+
+  }
+
+  &--focus {
+    outline: none;
+
+    #{$class-input} {
+      &-container {
+        border: 1px solid $c-primary;
+        @extend %sui-atom-input-input-focus;
```

Comment:
Extends should be placed first inside the selector. This way I'm pretty sure you're overwriting the `border` property you've defined above, but anyway, it's strange to extend placeholders at the bottom of a selector.